### PR TITLE
Add refund script

### PR DESCRIPTION
**Pull request by PJColombo**
This PR creates a script to refund all hatch contributors.

### CODE REVIEW

File: scripts/refund.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    currentContributorMsg = `Contributor ${account} (${i + 1} of ${contributors.length})`;
+
+    for (let j = 0; j < contributions.length; j++) {
+      const { vestedPurchaseId } = contributions[j];
+
+      spinner = spinner.start(
+        `${currentContributorMsg}: Refunding contribution ${j + 1} of ${contributions.length}...`
+      );
+
+      (await hatch.refund(account, vestedPurchaseId)).wait(2);
```
Comment (sembrestels) [inline scripts/refund.ts]: This probably should be `await (await hatch.refund(...)).wait(2);`

File: scripts/refund.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    spinner.succeed(
+      `${currentContributorMsg}: ${contributions.length} contribution${contributions.length > 1 ? "s" : ""} refunded`
+    );
+  }
+
+  spinner.stopAndPersist({ text: "All hatch tokens refunded!", symbol: "🙌" });
```
Comment (sembrestels) [inline scripts/refund.ts]: I would add a try/catch in order to detect if any transaction has failed, showing a message similar to the one you print when it succeeds, and change this last message by "All transactions have been sent".

**Review by sembrestels — COMMENTED** [review]
I added two request for changes, very good job!

File: scripts/refund.ts (hunk 1 above)
Comment (PJColombo) [reply scripts/refund.ts]: Yeah, you're right.

File: scripts/refund.ts (hunk 2 above)
Comment (PJColombo) [reply scripts/refund.ts]: I use the main's try/catch: 
https://github.com/CommonsSwarm/hatch-template/blob/355d1255a4730444029fa3b27cdb476bd53d8a2e/scripts/refund.ts#L74
When an error occurs the spinner fails showing the last message displayed on the terminal but with an error icon so you know where it failed

**Comment by sembrestels** [comment]
Good!

**Pull request merged**